e2e/utils/DriverHelper.ts

```diff
@@ -390,10 +390,14 @@ export class DriverHelper {
     }
 
     public async navigateTo(url: string) {
-        await this.driver.navigate().to(url);
+        await this.navigateToUrl(url);
         await this.waitURL(url);
     }
 
+    public async navigateToUrl(url: string) {
+        await this.driver.navigate().to(url);
```

Comment:
Looks like duplication of "navigateTo" method. Please delete the "navigateToUrl" method and use the "navigateTo" method